specification/maps/data-plane/Render/stable/2024-04-01/render.json

```diff
@@ -552,6 +558,10 @@
                 {
                   "value": "microsoft.imagery",
                   "description": "TilesetId contains a combination of satellite and aerial imagery."
```

Comment:
@miaxzhitong Please change the description to: "description": "TilesetId contains a combination of satellite or aerial imagery."